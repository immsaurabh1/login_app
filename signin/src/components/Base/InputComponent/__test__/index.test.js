import React from "react";
import { render, cleanup } from "@testing-library/react";
import Input from "../index.js";
import "@testing-library/jest-dom/extend-expect";
import ReactDOM from "react-dom";

afterEach(cleanup);
it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Input />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("matches snapshot", () => {
  const { container } = render(<Input value="value" />);
  expect(container).toMatchSnapshot();
});
