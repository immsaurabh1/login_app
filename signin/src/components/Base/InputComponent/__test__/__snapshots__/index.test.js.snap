// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    style="width: 100%;"
  >
    <input
      data-testid="input-cmp"
      style="padding: 8px; font-size: 16px; margin-bottom: 12px; width: 100%; box-sizing: border-box;"
      value="value"
    />
  </div>
</div>
`;
